src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1208,7 +1218,25 @@ void ImportStoreIndirect(TypeDesc type)
 
         void ImportThrow()
         {
-            throw new NotImplementedException();
+            var value = Pop();
+
+            var obj = _typeSystemContext.GetWellKnownType(WellKnownType.Object);
+            var exception = _typeSystemContext.GetWellKnownType(WellKnownType.Exception);
```

Comment:
The IL verification rules from ECMA-335 require the argument of `throw` instruction to be object. It does not need to be exception.